doc/TSCC.rst

```diff
@@ -217,13 +323,10 @@ To figure out why a job is queued use :code:`scontrol show job <your_job_number>
 
 Debugging jobs the OS killed
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-#. Look at the output file :code:`<log_dir>/<jobName>.o<jobID>`, the first line should contain the node
-   name. (e.g. :code:`Nodes: tscc-5-7`)
-#. ssh into the node (you can do this to any node, but if you run a large process the OS will kill you because
-   you have not been scheduled to that node)
-#. Scan the os logs for a killed process `dmesg -T | grep <jobid>`
-
-The OS normally kills jobs because you ran over your memory limit.
+#. Look at the standard output and standard error files. Any error messages should be there.
+#. ssh into the node. You can do this to any node, but if you run a large process the OS will kill you because you have not been scheduled to that node. You can figure out the name of the node assigned to your job using :code:`squeue` once the status of the job is "RUNNING".
```

Comment:
```suggestion
#. ssh into the node while the job is running. You can do this to any node, but if you run a large process the OS will kill you because you have not been scheduled to that node. You can figure out the name of the node assigned to your job using :code:`squeue` once the status of the job is "RUNNING".
```